### PR TITLE
Support all aria-current values for pagination

### PR DESCRIPTION
**Pull request by tomalec**
https://www.w3.org/TR/wai-aria-1.1/#aria-current
Don't let pages wrap lines.

### CODE REVIEW

File: uniform.css [hunk 1]
```diff
@@ -279,13 +279,31 @@ Pagination
...
+  border: 1px solid rgb(105, 206, 243);
+}
+.uni-pagination
+  > .uni-pages
+  > *:hover:not([aria-current]):not(:disabled):not([disabled]),
+.uni-pagination
+  > .uni-pages
+  > *:hover[aria-current="false"]:not(:disabled):not([disabled]) {
+  border: 1px solid rgb(105, 206, 243);
+}
```
Comment (warpech) [inline uniform.css]: Please change the hover effect to be the same as in `uni-flat-button`.

File: index.html [hunk 2]
```diff
@@ -474,7 +474,7 @@ <h2>Pagination</h2>
                     <li><code>.uni-next</code> - to a list of pages,</li>
                     <li><code>.uni-pages</code> - to wrap the "next", or "last" link,</li>
                     <li><code>[disabled]</code> - to disable a link, simply put "disabled" attribute,</li>
-                    <li><code>[aria-current="true"]</code> - to mark current page, use ARIA attribute.</li>
+                    <li><code>[aria-current="true"]</code> - to mark current item, use ARIA attribute (see <a href="https://www.w3.org/TR/wai-aria-1.1/#aria-current">the spec</a> for more values).</li>
```
Comment (warpech) [inline index.html]: It seems to me that our recommendation for pagination should be `[aria-current="page"]`

**Comment by warpech** [comment]
> Don't let pages wrap lines.

What do you mean by that?

File: index.html (hunk 2 above)
Comment (tomalec) [reply index.html]: Only if you use pagination of well.. pages, you may also paginate the products, then you should use `[aria-current="item"]`, you may paginate the steps in the process then you should use `[aria-current="step"]`, and so on.

File: uniform.css (hunk 1 above)
Comment (tomalec) [reply uniform.css]: done

**Comment by tomalec** [comment]
@warpech is it now good to go?

File: index.html (hunk 2 above)
Comment (warpech) [reply index.html]: I think pagination as a UI pattern is only for pages. If someone abuses this pattern for other uses, they should adapt the guidelines to their abuse case.

For steps, you use a wizard design pattern: https://uxplanet.org/wizard-design-pattern-8c86e14f2a38

But that's my opinion, I won't argue here.

**Comment by tomalec** [comment]
released in https://github.com/Starcounter/uniform.css/releases/tag/0.0.7

**Pull request merged**